src/toolkit/res_manip.cc

```diff
@@ -25,6 +27,7 @@ Material::Ptr Squash(std::vector<Material::Ptr> ms) {
   for (int i = 1; i < ms.size(); ++i) {
     m->Absorb(ms[i]);
   }
+  m->ChangePackageId();
```

Comment:
Is it possible to squash something into a package?  What if you have mat A that is filling the min of a its package and then you squash mat B that will fit in that package?  Can we/should we avoid removing the package?

Reply:
The way I have it is that squash always removes packaging. I don't feel strongly that we should keep the packaging, but I will change it if you think that's important